### PR TITLE
Add message to fixheader.exe target if Mono isn't installed

### PR DESCRIPTION
**Pull request by phrohdoh**
Hopefully this will help with issues like https://github.com/OpenRA/ra2/issues/301

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -291,6 +291,7 @@ server: $(server_TARGET)
 
 # Patches binary headers to work around a mono bug
 fixheader.exe: packaging/fixheader.cs
+	@todo_replace_with_csc || (echo "Mono is not installed. Please install Mono from http://www.mono-project.com/download/ before building OpenRA."; exit 1)
```
Comment (pchote) [inline Makefile]: `command -v $(CSC)`

Comment (Phrohdoh) [inline Makefile]: Thanks!

**Comment by pchote** [comment]
The tests break as expected, so :+1: once the proper check is added.

**Comment by Phrohdoh** [comment]
Updated.

File: Makefile [hunk 2]
```diff
@@ -291,6 +291,7 @@ server: $(server_TARGET)
 
 # Patches binary headers to work around a mono bug
 fixheader.exe: packaging/fixheader.cs
+	@command -v $(CSC) || (echo "Mono is not installed. Please install Mono from http://www.mono-project.com/download/ before building OpenRA."; exit 1)
```
Comment (Phrohdoh) [inline Makefile]: Blah, I was doing this previously but removed it because I thought you were implying you wanted the output.

**Comment by Phrohdoh** [comment]
Redirected to `/dev/null`.

**Review by pchote — APPROVED** [review]

**Comment by reaperrr** [comment]
This needs another approval from a non-Windows user, I think, @OpenRA/engine-hackers.

**Comment by pchote** [comment]
We don't have any other active non-windows @OpenRA/engine-hackers, so we'll have to take this without a second approval.

**Pull request merged**